pkg/consts/consts.go

```diff
@@ -28,4 +28,7 @@ const (
 	NetworkDataplaneAzure  = "azure"
 
```

Comment:
IMO, most of these should be in `options` to not have knowledge coupling.
But for now, let's keep it in one place for consistency.